pkg/core/writer/writer.go

```diff
@@ -230,10 +252,13 @@ func New(conf *config.WriterConfig, dpChan chan []*datapoint.Datapoint, eventCha
 	}
 	sw.spanWriter.Start(ctx)
 
-	log.Infof("Sending datapoints to %s", sw.client.DatapointEndpoint)
-	log.Infof("Sending events to %s", sw.client.EventEndpoint)
-	log.Infof("Sending trace spans to %s", sw.client.TraceEndpoint)
-
+	if conf.SignalFxAccessToken != "" {
+		log.Infof("Sending datapoints to %s", sw.client.DatapointEndpoint)
+		log.Infof("Sending events to %s", sw.client.EventEndpoint)
+		log.Infof("Sending trace spans to %s", sw.client.TraceEndpoint)
+	} else {
+		log.Infof("SignalFx client is deactivated, provide a token to enable")
```

Comment:
Should be enabled by providing the realm and/or ingest/event/trace urls.

Reply:
Alright, message is fixed.